src/main/resources/application.yaml

```diff
@@ -43,8 +43,8 @@ spring:
           schema-generation:
             create-source: metadata
             scripts:
-              action: none
-              #action: drop-and-create
+#              action: none
+              action: drop-and-create
```

Comment:
do not commit with drop and create, commit with none.

drop and create should only be local s and when required